snapshot-labs/sx-ui
# feat(starknet): add mainnet network

### Summary

Closes: https://github.com/snapshot-labs/sx-ui/issues/823

Depends on: https://github.com/snapshot-labs/mana/pull/121
Depends on: https://github.com/snapshot-labs/sx-api/pull/203

### How to test

1. Make sure has account set up on mainnet (run your own or wait for it to be updated on deployment)
2. Run sx-api from linked PR with `NETWORK_NODE_URL=https://starknet-mainnet.infura.io/v3/46a5dd9727bf48d4a132672d3f376146 NETWORK=SN_MAIN yarn dev`.
3. Update URLs for sx-api and mana if using local versions.
4. Run UI: `VITE_ENABLED_NETWORKS=sn yarn dev`
4. Deploy new space on Starknet.
